### ISSUE TITLE
Regarding the pretraining issue of abot_m0.

### ISSUE THREAD

**Issue by tracer0721** [amap-cvlab/ABot-Manipulation#37]
1. The dataset robocoin has been updated, so I will keep the consistent ones and delete the ones that cannot be found from the JSON file under dataloader/gr00t_lerobot/robot_type_map. Do I need to modify weights or other code? I pretrained and trained S2 on robotwin without modifying weights or other code, and the success rate was around 14%. I'm not sure if it's a problem with the pre training dataset or the pre training configuration. Has anyone encountered a similar problem before? 
2. Can the official pretraining weights be reproduced using examples/Pretrain/run_pretain.sh? If so, can you provide specific pre training configurations, such as batch (as mentioned in the paper, the pre training bs is 1024?), Gradient-ACccumulator_steps, framework.action_model.diffusion_model_cfg.output_dim in ABot_pretro. yaml.
